### PR TITLE
Add initial draft of Pyth bug bounty program

### PR DESCRIPTION
**Pull request by claudijd**
A draft proposal for adding a bug bounty program for Pyth.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,11 +1,41 @@
...
+
+### In Scope Assets
+- **[Pyth-Client](https://github.com/pyth-network/pyth-client)**
+
+_Note: We will be adding additional scope as we complete due diligence efforts for those scopes._
+### How to report a security vulnerability?
+
+Please do not file a GitHub Issue for security vulnerabilities.
+
+Please email security@pyth-network.com and provide your GitHub username and we will setup a draft security advisory for further discussion.
```
Comment (claudijd) [inline SECURITY.md:32]: Note, this alias is not yet live.

File: SECURITY.md [hunk 2]
```diff
@@ -1,11 +1,54 @@
...
+
+_Note: We will be adding additional scope as we complete due diligence efforts for those scopes._
+
+### Out of Scope
+
+- Anything involving social engineering or compromise of Pyth employees or infra
+- Network-based denial of service
+- Best practice critiques
+- Sybil attacks
+- Lack of liquidity
```
Comment (jayantk) [inline SECURITY.md:37]: not clear exactly what this means

File: SECURITY.md [hunk 3]
```diff
@@ -1,11 +1,54 @@
...
+    - Software flaws in the on-chain program that cause Pyth to publish an inaccurate price when ≥ 3/4 of the contributing publishers are accurate.
+    - Flaws enabling denial-of-service attacks for public-facing APIs
+    - Remote code execution
+    - Exposure of private keys for Pyth publishers or permissionless services.
+
+_Note: We do encourage vulnerability reporters to submit issues outside of the above mentioned payout structure, though we want to be clear that we will exercise discretion on a case by case basis whether an issue warrants a payout and what that payout would be._
+
+### In Scope Assets
+- **[Pyth-Client](https://github.com/pyth-network/pyth-client)**
+- **[Pyth2Wormhole](https://github.com/pyth-network/pyth2wormhole)**
```
Comment (ali-bahjati) [inline SECURITY.md:26]: We just renamed it to Pyth Crosschain

**Comment by claudijd** [comment]
Closing this PR, tenative plan is to have a pretty page on Pyth website as a the landing page rather than the market down, but still follow the Solana pattern of using GH Draft Advisories as the place where we collaborate with white-hat.

**Pull request closed without merging**